gradle/findbugs-exclude.xml

```diff
@@ -266,13 +266,6 @@ For a detailed description of findbugs bug categories, see http://findbugs.sourc
 
     <Match>
         <!-- TODO: fix this (see KAFKA-4996) -->
-        <Class name="org.apache.kafka.streams.state.internals.Segments"/>
```

Comment:
@guozhangwang Thanks for reviewing. I did not remove it since I was not sure why was it present there at the first place.